torchgen/model.py

```diff
@@ -1026,18 +1026,14 @@ def schema_order_arguments(self) -> Iterator["Argument"]:
             self.arguments.out,
         )
 
+    decl_re = re.compile(r"(?P<name>[^\(]+)\((?P<args>.*)\) -> (?P<returns>.*)")
```

Comment:
@bdhirsh are you ok with this?  you should probably take a look at the other codegen changes in this PR and see if you have any issues with it.

Reply:
The original error messages were a little more specific. But I think this is ok